frontend/packages/jobs/src/functions/processGenerateReview.ts

```diff
@@ -74,26 +55,13 @@ export const processGenerateReview = async (
 
     // Filter out null values and join content
     const docsContent = docsContentArray.filter(Boolean).join('\n\n---\n\n')
-    const prompt = PromptTemplate.fromTemplate(REVIEW_TEMPLATE)
 
-    const model = new ChatOpenAI({
-      temperature: 0.7,
-      model: 'gpt-4o-mini',
-    })
-
-    const chain = prompt.pipe(model)
-    const response = await chain.invoke(
-      {
-        docsContent,
-        schemaFiles: payload.schemaFiles,
-        schemaChanges: payload.schemaChanges,
-      },
-      {
-        callbacks: [langfuseLangchainHandler],
-      },
```

Comment:
Sorry, missing it

Reply:
fixing https://github.com/liam-hq/liam/pull/1058/commits/67f3ec735fcb85f9b48a41a41f69138566e983dc